package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "git-user-name",
   "description": "Get a user's name from git config at the project or global scope, depending on what git uses in the current context.",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Please revert this, versions are typically bumped as part of the release process. Thanks

Reply:
Done.